### ISSUE TITLE
Show error message if plugin cannot be enabled

### ISSUE THREAD

**Issue by darrylhein** [wildbit/postmark-wordpress#74]
After a good 30+ minutes, we finally found the FAQ item stating that the plugin is not shown if another plugin is using `wp_mail`. Would it instead be possible to still show the menu item with a message stating why it's not enabled/being used?

**Comment by pgraham3** [comment]
@darrylhein Yes this should definitely be improved - sorry for the trouble there. I think preventing the plugin's activation altogether if a conflict is detected may be the best option for ensuring folks don't try to use it while another plugin is also overriding `wp_mail`.
